### PR TITLE
feat(rviz): visualize error diag graph

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

launcher change for https://github.com/autowarefoundation/autoware.universe/pull/9421
The error diag will be visualized on Rviz.

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by takayuki5168** [comment]
@isamu-takagi 
Please approve this PR if the changes look good to you.

File: autoware_launch/rviz/autoware.rviz [hunk 1]
```diff
@@ -268,21 +268,21 @@ Visualization Manager:
...
           Top: 10
           Topic:
-            Depth: 5
+            Depth: 1
             Durability Policy: Volatile
             History Policy: Keep Last
             Reliability Policy: Reliable
-            Value: /system/emergency/hazard_status
-          Value: false
+            Value: /logging_diag_graph/debug/error_graph
```
Comment (isamu-takagi) [inline autoware_launch/rviz/autoware.rviz]: https://github.com/autowarefoundation/autoware.universe/pull/9421#discussion_r1877171488
```suggestion
            Value: /logging_diag_graph/debug/error_graph_text
```

Comment (takayuki5168) [reply autoware_launch/rviz/autoware.rviz]: Oops. Thank you!

**Review by isamu-takagi — APPROVED** [review]
LGTM

**Pull request merged**